tests/test_ingest.py

```diff
@@ -318,6 +318,48 @@ def test_mtx_bundle_argument_validation(self):
 
         self.assertRaises(ValueError, self.setup_ingest, args)
 
+    def test_ingest_loom(self):
+        """Ingest Pipeline should extract and transform loom files
+        """
+
+        args = [
+            "--study-accession",
+            "SCP1",
+            "--file-id",
+            "1234abc",
+            "ingest_expression",
+            "--taxon-name",
+            "Homo Sapiens",
```

Comment:
Also updated in mock data.

```suggestion
            "Homo sapiens",
```

Reply:
Addressed in f708656